docs/source/documentation/team/wow-2.html.md.erb

```diff
@@ -0,0 +1,222 @@
+- **Capacity Estimation:** Estimate the capacity of the team for the sprint, and assess whether the tickets brought in match that.
+- **Sprint Backlog:** Select the stories that will be brought into the sprint, and assign them to team members where appropriate.
+
+**Output:**
+
+- Ensure everyone has stories that they feel they can work on
+- Ensure everyone has a shared understanding of the tasks to deliver this sprint
+
+#### Audience
+
+Entire DPAT team
+
+### Sprint Review
+
+- Review completed work with stakeholders, gathering feedback for future sprints. Usually encorporated with Planning.
+
+#### Audience
+
+DPAT team
+
+### Sprint Retrospective
+
+- Reflect on the sprint’s process, celebrating successes and identifying opportunities for improvement
+- Review the process, not the stories
+- Define how we could improve
+- Share kudos and feedback
+
+#### Audience
+
+DPAT team
+
+### Show and Tell
+
+- At the end of each sprint we have a thirty-minute show and tell showcase to share with our colleagues and the wider organisation our achievements during the sprint. At the end of each sprint we have a thirty-minute show and tell showcase to share with our colleagues and the wider organisation our achievements during the sprint. The showcase normally consists of an introduction summing up the sprint and if we met our sprint goal followed by demonstration and then what’s next in the road map
+
+#### Audience
+
+All of Data Platform and wider audience
+
+## GitHub Practices
+
+- We utilise GitHub for managing code, documentation, and support, with GitHub Projects for tracking work and GitHub pull requests for reviewing and merging code
+- When working in our repos, please adhere to GitHub housekeeping guidelines when creating pull requests, ensuring clear titles and descriptions, linking to issues when applicable, and requesting reviews upon readiness
+
+### GitHub Housekeeping
+
+- When creating a pull request, ensure you have a clear title and description
+- If applicable, link your pull request to an issue with the special keywords such as `closes`, `fixes` or `resolves` [GitHub documentation for linking pull requests to issues](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue)
+- If your pull request is not ready for review, convert it to a draft
+- When your pull request is ready for review, remove the draft status and ensure all checks are passing
+- When all checks are passing, use the `/pull` command in [`#ask-data-platform`](https://mojdt.slack.com/archives/C04VBJTKY58) to request a review
+
+## Development Practices
+
+- This is quite a large topic, so we maintain a separate page for it: [Developing](/documentation/platform/infrastructure/developing.html)
+
+## Support
+
+- Each member of the team is included in the support rota
+- Support cover is for one day (10:00-16:30) as allocated by the rota
```

Comment:
Are these times accurate? I assumed it was 9-5, and think thats what pager duty uses
```suggestion
- Support cover is for one day (09:00-17:00) as allocated by the rota
```